database/migrations/2023_09_20_125637_create_game_ref_table.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('game_ref', function (Blueprint $table) {
```

Comment:
only change I will do is the name of the table
```suggestion
        Schema::create('game_user', function (Blueprint $table) {
```
also you can create a seeder for the first list of games that we will support